### DISCUSSION TITLE
Nice WebSite

### DISCUSSION THREAD

**Discussion by tormentr** [solodit/feedback#2] [New Feature Requests]
When $Open \ Source$ ?

**Comment by PatrickAlphaC** [comment]
Thanks for this!

Currently, we're in a state of continuous evolution, which necessitates centralized management of our operations. This allows us to concentrate on our roadmap and frequent shifts in strategy, making it challenging for us to present our code in a comprehensible manner to external parties. 

However, we're deeply committed to embracing an open-source approach as much as possible and have already initiated the process by open-sourcing certain elements, such as tags and categories.

Additionally, our codebase has a lot of sensitive information in it at the moment, and we need to do some work to refactor it out.

**Reply by tormentr** [reply]
the whole site is done with Tailwind and nextjs , and what appears to be data being fetched from repos of published reports , what is this $confidential$  info are u referring to exactly ?

**Reply by PatrickAlphaC** [reply]
a lot of our hosting information is intermingled with the codebase

**Reply by tormentr** [reply]
ok , nice resource BTW !
